<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConfigWidget</class>
 <widget class="QWidget" name="ConfigWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>469</width>
    <height>357</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true">Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="labelDate">
       <property name="text">
        <string>Date:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QComboBox" name="comboBoxDate">
         <property name="sizeAdjustPolicy">
          <enum>QComboBox::AdjustToContents</enum>
         </property>
         <item>
          <property name="text">
           <string>Last modification</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Last access</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Creation</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Current</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Custom</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QDateTimeEdit" name="dateTimeEditCustom">
         <property name="buttonSymbols">
          <enum>QAbstractSpinBox::NoButtons</enum>
         </property>
         <property name="calendarPopup">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="labelFormat">
       <property name="text">
        <string>Format:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QComboBox" name="comboBoxFormat">
         <property name="sizeAdjustPolicy">
          <enum>QComboBox::AdjustToContents</enum>
         </property>
         <item>
          <property name="text">
           <string>Default</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>ISO date</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Custom</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="comboBoxCustomFormat">
         <property name="editable">
          <bool>true</bool>
         </property>
         <item>
          <property name="text">
           <string notr="true">yyyy-MM-dd HH:mm:ss</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string notr="true">yyyy-MM-dd</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string notr="true">HH:mm:ss</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="toolButtonFormatHelp">
         <property name="text">
          <string>?</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="labelSeparator">
       <property name="text">
        <string>Separator:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="lineEditSeparator"/>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="labelPosition">
       <property name="text">
        <string>Position:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QGroupBox" name="groupBoxInsertPosition">
       <property name="title">
        <string/>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
       <layout class="QGridLayout" name="gridLayout">
        <property name="margin">
         <number>0</number>
        </property>
        <item row="0" column="0">
         <widget class="QRadioButton" name="radioButtonPrefix">
          <property name="text">
           <string>Prefix</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QRadioButton" name="radioButtonSuffix">
          <property name="text">
           <string>Suffix</string>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QRadioButton" name="radioButtonCustomPosition">
          <property name="text">
           <string>Custom position:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="3">
         <widget class="QSpinBox" name="spinBoxCustomPosition"/>
        </item>
        <item row="0" column="4">
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>388</width>
       <height>215</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
